### DISCUSSION TITLE
🔥 Working with outside collaboratos

### DISCUSSION THREAD

**Discussion by joshoua-bigler** [community/community#8835] [Projects and Issues] [answered]
Working with outside collaborators within GitHub Project Beta should be possible!

**Comment by azenMatt** [comment, marked as answer]
Hi @JoshouaDPE - we have been working on this and have a beta version we are testing, let me know if you (or anyone else reading) would like to help us test this before we roll it out to everyone shortly. 

If you can let me know your org name, I will add it to the feature flag 😄

**Reply by joshoua-bigler** [reply]
Hi @azenMatt 
I would like to help you test the function👍 
The name of my organization is SteinemannDPE.

**Reply by azenMatt** [reply]
Added :) 

As a heads up - this list of possible outside collaborators is anyone who is already a collaborator on a repo. Let me know how it goes!

**Reply by joshoua-bigler** [reply]
Hi @azenMatt 
Everything is going well so far👍 But I have one more point.
It would be nice if the respective outside collaborator could also be assigned to a task although it is not yet assigned to a repository.
